reactome_base_sonatype/pom.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.reactome.base</groupId>
+    <artifactId>reactome-base</artifactId>
+    <packaging>jar</packaging>
+    <version>1.2.22-SNAPSHOT</version>
+    <name>Reactome Base</name>
+    <description>Reactome Java API</description>
+    <url>https://reactome.org</url>
+
+    <licenses>
+        <license>
+            <name>GNU AFFERO GENERAL PUBLIC LICENSE</name>
+            <url>https://www.gnu.org/licenses/agpl-3.0.txt</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Reactome Team</name>
+            <email>help@reactome.org</email>
+            <organization>Reactome</organization>
+            <organizationUrl>https://reactome.org</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/reactome/CuratorTool.git</connection>
+        <developerConnection>scm:git:ssh://github.com/reactome/CuratorTool.git</developerConnection>
+        <url>https://github.com/reactome/CuratorTool</url>
+        <tag>v1.2.21</tag>
```

Comment:
The tag version for curator tool is 1.2.21 here, but 1.2.22 for the version? I'm sure this wasn't missed, I'm just curious about the discrepancy

Reply:
You are correct!  I didn't update the tag for the new version.  I will fix it - thanks!

Reply:
Fixed!